modules/topics/client/config/topics.client.config.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+angular.module('topics').run(['Menus',
+  function (Menus) {
+    // Set top bar menu items
+    Menus.addMenuItem('topbar', {
+      title: 'Topics',
+      state: 'topics',
+      position: 2,
+      roles: ['*']
+    });
+  }
+]);
```

Comment:
Adds a new menu for the Topic creation/update UI. There's nothing there yet though - maybe I should remove this from this PR?